bundles/org.openhab.binding.vdr/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,213 @@
+				<label>Current EPG Event Begin</label>
+			</channel>
+			<channel id="currentEventEnd" typeId="vdrEventEnd">
+				<label>Current EPG Event End</label>
+			</channel>
+			<channel id="currentEventDuration" typeId="vdrEventDuration">
+				<label>Current EPG Event Duration in Minutes</label>
+			</channel>
+			<channel id="nextEventTitle" typeId="vdrEventTitle">
+				<label>Next EPG Event Title</label>
+			</channel>
+			<channel id="nextEventSubTitle" typeId="vdrEventSubTitle">
+				<label>Next EPG Event Sub Title</label>
+			</channel>
+			<channel id="nextEventBegin" typeId="vdrEventBegin">
+				<label>Next EPG Event Begin</label>
+			</channel>
+			<channel id="nextEventEnd" typeId="vdrEventEnd">
+				<label>Next EPG Event Begin</label>
+			</channel>
+			<channel id="nextEventDuration" typeId="vdrEventDuration">
+				<label>Next EPG Event Duration in Minutes</label>
+			</channel>
+		</channels>
+
+		<properties>
+			<property name="version">VDR Version</property>
+		</properties>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Hostname / IP Address</label>
+				<description>Hostname or IP Address of VDR instance</description>
+			</parameter>
+			<parameter name="port" type="integer" required="false">
+				<label>SVDRP Port</label>
+				<description>SVDRP Port of VDR instance</description>
+				<default>6419</default>
+			</parameter>
+			<parameter name="refresh" type="integer">
+				<label>Refresh Interval</label>
+				<description>Interval in seconds the data from VDR instance is refreshed</description>
+				<advanced>true</advanced>
+				<default>30</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="vdrMessage">
+		<item-type>String</item-type>
+		<label>Display Message</label>
+		<description>Send Message to be displayed on VDR</description>
+		<state readOnly="false"/>
+	</channel-type>
+	<channel-type id="vdrChannel">
+		<item-type>Number</item-type>
+		<label>Channel Number</label>
+		<description>Current Channel Number</description>
+		<state readOnly="false" pattern="%d"/>
+	</channel-type>
+	<channel-type id="vdrChannelName">
+		<item-type>String</item-type>
+		<label>Channel Name</label>
+		<description>Current Channel Name</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="vdrRecordingActive">
+		<item-type>Switch</item-type>
+		<label>Recording Active</label>
+		<description>Is currently a Recording Active?</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="vdrVolume">
+		<item-type>Dimmer</item-type>
+		<label>Volume</label>
+		<description>Current Volume</description>
+		<state min="0" max="100" step="1" pattern="%d %%"/>
+	</channel-type>
+	<channel-type id="vdrPower">
+		<item-type>Switch</item-type>
+		<label>Power State</label>
+		<description>Power State (to switch off VDR)</description>
+		<state readOnly="false"/>
+	</channel-type>
```

Comment:
There are predefined channel types you can use. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types This could replaced by system.power.